perl_modules/Atlas/ZoomaClient.pm

```diff
@@ -244,6 +251,80 @@ sub _query_zooma {
 }
 
 
+sub zooma_ontology_lookup {
+    my ( $filename ) = @_;
+    my $abs_path = dirname(File::Spec->rel2abs(__FILE__));
+    my $zoomaOntologyLookupFile = "$abs_path/../../supporting_files/". $filename;
+    open (my $in_fh, '<', $zoomaOntologyLookupFile) or die $!;
+    my %ontology_lookup;
+    while ( my $line = <$in_fh> ) {
+        my ( $property_type, $organism, $ontologies ) = split /\t/, $line;
+        $ontology_lookup{$property_type}{$organism} = $ontologies;
+    }   
+    return \%ontology_lookup;
+}
+
+sub get_plants_species {
+    my ( $rnaseqer_api ) = @_;
+    my $plants_query=$rnaseqer_api . '/tsv/0/getOrganisms/plants';
```

Comment:
Do we have any alternative to retrieve plant species from somewhere else instead of rnaseqer? To reduce ties to it.